src/app/i18n/ru.ts

```diff
@@ -35,9 +35,14 @@ export const Russian = {
     randomImageHint: 'Показать случайный скриншот',
     randomizeGalleryDescription: 'Рандомизирует порядок видеофайлов после каждого поиска',
     randomizeGalleryHint: 'Рандомизировать порядок галереи',
+    // TODO: Translate these
```

Comment:
Thank you for the `TODO` -- I'll get these translated.
My dirty trick is that I simply used Google Translate :trollface: 
Though I am fluent in Russian so it's easy to confirm the translations aren't bad 👍